faq/faq.md

```diff
@@ -0,0 +1,351 @@
+valuable to reach a consensus about this within your project.
+
+## How do I deal with a file that has been edited by many people? {#many-copyright-statements}
+
+Some files are edited by many people and would have an extremely long list of
+copyright holders in the header. This may be aesthetically unpleasing, but is
+not incorrect.
+
+If you would rather not deal with having so many copyright notices, some
+projects such as Chromium circumvent this problem by using "Copyright (c) 2013
+The Chromium Authors" as their copyright tag. You may consider doing this, but
+you should keep a list of authors in your project.
+
+## Why can't I just use version control to record copyright? {#vcs-copyright}
+
+In [a previous question](#copyright-holder-author), we distinguished between
+copyright holders and authors, which are not always the same. Version control
+typically only records authorship, which makes it unsuitable for the task of
+recording copyright.
+
+Another obstacle is that version control history may contain errors, and fixing
+such an error would require rewriting the history, causing all contributors to
+have to re-download the new trunk.
+
+## Can I use DEP5 to declare licensing and copyright? {#dep5}
+
+See next question.
+
+## Can I bulk-license whole directories? {#bulk-license}
+
+DEP5
+
+## Can I license only a part of a file as being under a different license? {#partial-license}
+
+The short answer is that yes, you can, but no, there is no standard way for
+REUSE to recognize this. If you have a small segment of a file that is licensed
+differently, you should list that license under a separate
+`SPDX-License-Identifier` tag in the header.
+
+You can use your own comments to specify which segment is separately licensed.
+For instance: "The class Foo is copied from project Bar and licensed under MIT.
+All changes are licensed under GPL-3.0-or-later."
+
+A possible way to circumvent the problem is to extract the segment from the
+file, and to keep it in its own file.
+
+## How do I properly declare multi-licensing? {#multi-licensing}
+
+You should always include all licenses in the `LICENSES/` directory.
+
+The correct SPDX license expression that applies to the file depends on the
+intent. If all the code within is licensed under multiple licenses, and the
+licensee can choose under which license they consume the work, use
+`SPDX-License-Identifier: MPL-1.1 OR GPL-2.0-or-later OR LGPL-2.1-or-later`, as
+parts of Firefox do.
+
+If all the code within the file is licensed under multiple licenses, and the
+user must comply with all licenses simultaneously, use
+`SPDX-License-Identifier: LGPL-2.0-or-later AND AML`, as can be found in Simple
+DirectMedia Layer (SDL).
+
+If all the code within the file is licensed under either one license or another
+(for instance, all code is under GPL-2.0-only, but one function is under MIT),
+use separate tags `SPDX-License-Identifier: GPL-2.0-only` and
+`SPDX-License-Identifier: MIT`.
```

Comment:
This is effectively the same as `SPDX-License-Identifier: GPL-2.0-only AND MIT`.

Reply:
See https://github.com/spdx/spdx-spec/issues/123